addons/web/static/src/views/kanban/kanban_renderer.js

```diff
@@ -392,6 +392,10 @@ export class KanbanRenderer extends Component {
         );
     }
 
+    canApplyExampleData() {
+        return !!this.exampleData && this.exampleData.examples.length && this.canCreateGroup();
+    }
+
```

Comment:
Hello @RaoufGhrissi 

A topic we did not address in #115909 is the prop `canApplyExampleData` you add to the `KanbanColumnQuickCreate`.

The alternative I propose is the `canShowExamples` method.
With it, all you have to do is to override this function in `forecast_kanban_column_quick_create.js`.
Does this fit your usecase ?

Reply:
hey @brboi and i will see how can i replace this part https://github.com/odoo/odoo/blob/b5de4a7840916a0c2e155e5c0677e6879f64e7ed/addons/project/static/src/views/project_task_kanban/project_task_kanban_renderer.js#L46-L52 by the same logic and ask you for help if needed or confirm if it fit

Reply:
hey @brboi normally yes, my bad i was copying some parts of my code and erased it. i will test and confirm to you

Reply:
fixed, i just overrided it `project_task_kanban_column_quick_create.js`
for `forecast_kanban_column_quick_create.js` it will be evaluated to false as we don't have examples so no need